validator/pom.xml

```diff
@@ -94,23 +94,30 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>3.1.1</version>
+                <artifactId>maven-war-plugin</artifactId>
```

Comment:
this change isn't related to what's in the commit message

Reply:
If this should be a question of yours, the manifest of the WAR file can only be adjusted by the renamed plugin.
I desired to trigger the WAR by command line similar to the JAR.